### DISCUSSION TITLE
[Feature Request] TTS override ( pronunciation )

### DISCUSSION THREAD

**Discussion by ManekiNeko1982** [vrcx-team/VRCX#983] [Ideas]
I'm visually impaired so make heavy use of vrcx's tts for notifications but I would love the ability to enter alternate names for friends for tts specifically so instead of vrcx saying example " under score UserName under score one thousand two hundred thirty four has joined" ( for _UsweName 1234 ) i can add a nick or over ride to just "User Name"

**Comment by SalbugVR** [comment]
I could see the TTS being used to read the first line of VRCX memo if used for an alt name like so.

![image](https://github.com/user-attachments/assets/c47ba65c-613e-4fca-b34f-5663de1dc25b)

**Reply by Natsumi-sama** [reply]
This feature is now in latest nightly build.

**Reply by ManekiNeko1982** [reply]
thank you
